test-extra/Makefile

```diff
@@ -26,6 +26,7 @@ FIND_ARGS= \
 	-path code/dune/vendor -not -prune -or \
 	-path code/dune/otherlibs -not -prune -or \
 	-path code/dune/example -not -prune -or \
+	-path code/duniverse/duniverse -not -prune -or \
```

Comment:
not used yet

Reply:
I kept it here so uncommenting the projects below works. I changed so this is easier to declare there.